### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by zayKenyon**
Re-wrote the README to incorporate feedback by @ririxidev and @Soneliem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,85 @@
...
-- If VALORANT is already running, launch the program and the presence will start
+ - If VALORANT is already running, launch the program and the presence will start
+
+
+ 
+## Support
+
+ Either make an issue or:
+
+ [![Discord][discord-shield]][discord-url]
```
Comment (Soneliem) [inline README.md]:
```suggestion
[![Discord Banner 2][discord-banner]][discord-url]
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,85 @@
...
+ - Run the program instead of launching VALORANT
      - If VALORANT is not running, the program will launch it for you
-- If VALORANT is already running, launch the program and the presence will start
+ - If VALORANT is already running, launch the program and the presence will start
+
+
+ 
+## Support
+
+ Either make an issue or:
```
Comment (Soneliem) [inline README.md]:
```suggestion
 Either make an issue or:  
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,85 @@
...
+
+ This project is not affiliated with Riot Games or any of its employees and therefore does not reflect the views of said parties. This is purely a fan-made project to     enhance VALORANT's skin inventory management.
+
+ Riot Games does not endorse or sponsor this project. Riot Games, and all associated properties are trademarks or registered trademarks of Riot Games, Inc.
+ 
+ 
+ 
+[discord-shield]: https://img.shields.io/discord/860288779558715402?color=7289da&label=Support&logo=discord&logoColor=7289da&style=for-the-badge
+[discord-url]: https://discord.gg/uGuswsZwAT
+
```
Comment (Soneliem) [inline README.md]:
```suggestion
[discord-banner]: https://discordapp.com/api/guilds/860288779558715402/widget.png?style=banner2

```

**Review by Soneliem — CHANGES_REQUESTED** [review]
As discussed earlier: Replace second discord shield with banner

**Comment by zayKenyon** [comment]
> As discussed earlier: Replace second discord shield with banner

Done 👍

**Comment by ririxidev** [comment]
There are still things that I requested to change before.

**Comment by zayKenyon** [comment]
> There are still things that I requested to change before.

If I have mistakenly left them out, please propose them as a change so I can commit them :)

File: README.md [hunk 4]
```diff
@@ -1,21 +1,85 @@
...
+ Valorant RPC (Rich Presence) allows you to show in-game details such as the current score in your Discord Profile! Additional features include showing current map, agent, idle status, etc.
+
+ 
+ <a>
+    <img src="assets/Demo1.png" alt="Demo" width="205" height="112">
+    <img src="assets/Demo2.png" alt="Demo" width="205" height="112">
 
-# Installation
-- Download the latest [release](https://github.com/colinhartigan/valorant-rpc/releases/) and run it.
+ </a>
```
Comment (ririxidev) [inline README.md:33]: no need for <a> tag, if you want to redirect to url, you need to <a> tag every image, not whole stuff

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by colinhartigan** [comment]
most heated pull request ive ever seen

**Review by EMREOYUN — APPROVED** [review]
In its current status, seems well-made.

**Comment by Soneliem** [comment]
🎉🎉🎉 Finally, over a year later

**Comment by colinhartigan** [comment]
i think it's been simmering for long enough now

**Pull request merged**